Sources/swift-format/Frontend/Frontend.swift

```diff
@@ -223,11 +223,26 @@ class Frontend {
           "Unable to read configuration for \(swiftFileURL.path): \(error.localizedDescription)")
         return nil
       }
+    } else {
+      // If reading from stdin and no explicit configuration file was given,
+      // walk up the file tree from the cwd to find a config.
+
+      let cwd = URL(FileManager.default.currentDirectoryPath)
```

Comment:
I think this needs to be `URL(fileURLWithPath: ...)`? I don't see an unlabeled `URL.init` documented anywhere, and it doesn't build on my system (Swift 5.9.2, macOS).

Reply:
Oops, let me fix that. No idea why I didn't catch that.